package.json

```diff
@@ -57,76 +58,44 @@
     "@babel/preset-env": "7.4.5",
     "@babel/preset-flow": "7.0.0",
     "@babel/preset-react": "7.0.0",
-    "acorn": "6.1.1",
+    "@redhat-cloud-services/frontend-components-config": "0.0.8",
     "axios": "0.19.0",
     "babel-core": "7.0.0-bridge.0",
     "babel-eslint": " 10.0.1",
     "babel-jest": "24.8.0",
     "babel-loader": "8.0.6",
     "babel-plugin-dual-import": "1.2.1",
     "babel-plugin-lodash": "3.3.4",
-    "clean-webpack-plugin": "3.0.0",
-    "copy-webpack-plugin": "5.0.3",
-    "cross-fetch": "3.0.3",
-    "css-loader": "2.1.1",
-    "d3": "5.9.2",
-    "enzyme": "3.9.0",
-    "enzyme-adapter-react-16": "1.13.2",
+    "enzyme": "3.10.0",
+    "enzyme-adapter-react-16": "1.14.0",
     "enzyme-to-json": "3.3.5",
     "eslint": "5.16.0",
     "eslint-config-prettier": "4.3.0",
-    "eslint-loader": "2.1.2",
     "eslint-plugin-react": "7.13.0",
-    "file-loader": "3.0.1",
-    "git-revision-webpack-plugin": "3.0.3",
-    "glob": "7.1.4",
-    "html-replace-webpack-plugin": "2.5.5",
-    "html-webpack-plugin": "3.2.0",
     "identity-obj-proxy": "3.0.0",
     "jest": "24.8.0",
-    "lodash-webpack-plugin": "0.11.5",
-    "material-ui": "0.20.2",
-    "mini-css-extract-plugin": "0.7.0",
     "moment": "2.24.0",
-    "node-sass": "4.12.0",
     "npm-run-all": "4.1.5",
     "prop-types": "15.7.2",
-    "react": "16.8.6",
-    "react-content-loader": "4.2.1",
-    "react-dom": "16.8.6",
-    "react-markdown": "4.0.8",
-    "react-redux": "5.1.0",
-    "react-router-dom": "4.3.1",
     "react-test-renderer": "16.8.6",
-    "react-timeago": "4.4.0",
-    "redux": "4.0.1",
-    "redux-logger": "3.0.6",
-    "redux-promise-middleware": "6.1.0",
-    "regenerator-runtime": "0.13.2",
-    "sass-loader": "7.1.0",
     "seamless-immutable": "7.1.4",
-    "source-map-loader": "0.2.4",
-    "style-loader": "0.23.1",
     "stylelint": "10.0.1",
     "stylelint-config-recommended-scss": "3.3.0",
-    "stylelint-scss": "3.7.0",
-    "topojson": "3.0.2",
-    "webpack": "4.32.2",
+    "stylelint-scss": "3.8.0",
+    "webpack": "4.33.0",
     "webpack-bundle-analyzer": "3.3.2",
     "webpack-cli": "3.3.2",
-    "webpack-dev-server": "3.4.1",
-    "webpack-visualizer-plugin": "0.1.11",
-    "write-file-webpack-plugin": "4.5.0"
+    "webpack-dev-server": "3.7.0"
   },
   "scripts": {
     "build": "webpack --config config/prod.webpack.config.js",
     "test": "jest --passWithNoTests",
     "lint": "npm-run-all lint:*",
     "lint:js": "eslint config src",
     "lint:sass": "stylelint 'src/**/*.scss' --config .stylelintrc.json",
-    "prod": "NODE_ENV=production webpack-dev-server",
+    "prod": "NODE_ENV=production webpack-dev-server --config config/dev.webpack.config.js",
```

Comment:
prod and dev use the same webpack config?

Reply:
That is prod start, so the files are minified and you can check live reloads as if app was deployed. he actual build happens in `build` script

Reply:
yep! the prod and dev webpack config are exactly the same, could maybe get rid of one of em for giggles? 🤷‍♂ (but secretly i don't want to dismiss all this great review)

Reply:
It's essentially emulating prod, but locally. This doesn't actually build what will be live on public prod.